src/test/java/com/epam/tc/hw4/steps/FirstExerciseSteps.java

```diff
@@ -0,0 +1,106 @@
+package com.epam.tc.hw4.steps;
+
+import com.epam.tc.hw4.components.BenefitComponent;
+import com.epam.tc.hw4.components.LoginComponent;
+import com.epam.tc.hw4.pages.HomePage;
+import com.epam.tc.hw4.tests.BaseTest;
+import io.qameta.allure.Step;
+import java.util.List;
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.WebDriver;
+
+public class FirstExerciseSteps extends BaseTest {
```

Comment:
Let's make something like ActionSteps and AssertionSteps classes.
Not FirstExerciseSteps and SecondExerciseSteps.

Reply:
Make necessary changes.